canonical-web-and-design/ubuntu.com
# Update the automotive page

## Done
Update the automotive page

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/automotive
- Check that the content matches [the copy doc](https://docs.google.com/document/d/1PVliB92h4ZqexaZV9F5MIcX7aRG6yBAGH1hG7x8UZTw/edit#)
- Check it matches [the design](https://user-images.githubusercontent.com/65166314/87928171-47f1b600-ca7c-11ea-9cc4-45bba031ed84.png)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2946
